settings.gradle

```diff
@@ -65,6 +65,7 @@ include(
     "zgw:catalogi-api",
     "zgw:documenten-api",
     "zgw:notificaties-api",
+    "zgw:object-management",
```

Comment:
This new module should also be added as a dependency to `/app/gzac/build.gradle`